### DISCUSSION TITLE
How do you show a livewire component inside a model?

### DISCUSSION THREAD

**Discussion by darren-glanville** [filamentphp/filament#10407] [Help]
I have a page where it would be useful to show a calendar that people can open as they are creating a booking. 

How do I go about showing this is a model that can slide in on the page and then be reopened as needed?

**Comment by leandrocfe** [comment]
inside a model or modal?

**Comment by darren-glanville** [comment]
A modal

On Wed, Dec 20, 2023 at 12:03 PM Leandro Costa Ferreira <
***@***.***> wrote:

> inside a model or modal?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/filamentphp/filament/discussions/10407#discussioncomment-7907532>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APOL47W4OJIGA32NNADHA4LYKLHZTAVCNFSM6AAAAABA4L5DN6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TSMBXGUZTE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Darren Glanville*



www.darren-glanville.dev
***@***.***
07477885121

**Reply by leandrocfe** [reply]
You could use a modal action
https://filamentphp.com/docs/3.x/actions/modals#modal-forms

You could use `->modalContent(view('custom-view'))` method to render a view file and render a livewire component inside the view

```html
<div>
@livewire('custom-livewire-component')
</div>
```
